hardhat.config.ts

```diff
@@ -46,6 +46,7 @@ const config: HardhatUserConfig = {
     } as SSVNetworkConfig,
     hardhat: {
       allowUnlimitedContractSize: true,
+      gas: 5000000,
```

Comment:
I had problems using hardhat network when bulk registering validators. With less than 10 validators I reached 30M gas, which is not the real cost. I set this param to fix it.

Reply:
What does this do? Why was reaching so high?  WIth this param that you added is the gas correct now?  

Not related but the param above it "allowUnlimitedContractSize" i think maybe its best we put that back to true since were diamond now?

Reply:
Using default hardhat config, for some reason moving from bulk registering 8 validators (less than 1M gas) to 10 results in more than 30M gas. Specifying the gas in the network config results in real numbers close to holesky ones.
Agree on removing `allowUnlimitedContractSize`